x-pack/plugins/upgrade_assistant/__jest__/client_integration/es_deprecations/deprecations_list.test.ts

```diff
@@ -63,16 +63,16 @@ describe('ES deprecations table', () => {
     const reindexDeprecation = esDeprecationsMockResponse.deprecations[3];
 
     // Since upgradeStatusMockResponse includes ML and reindex actions (which require fetching status), there will be 3 requests made
-    expect(server.requests.length).toBe(totalRequests + 3);
-    expect(server.requests[server.requests.length - 3].url).toBe(
+    expect(server.requests.length).toBe(totalRequests + 4);
+    expect(server.requests[server.requests.length - 4].url).toBe(
       `${API_BASE_PATH}/es_deprecations`
     );
-    expect(server.requests[server.requests.length - 2].url).toBe(
+    expect(server.requests[server.requests.length - 3].url).toBe(
       `${API_BASE_PATH}/ml_snapshots/${(mlDeprecation.correctiveAction as MlAction).jobId}/${
         (mlDeprecation.correctiveAction as MlAction).snapshotId
       }`
     );
-    expect(server.requests[server.requests.length - 1].url).toBe(
+    expect(server.requests[server.requests.length - 2].url).toBe(
```

Comment:
Do we also want to verify what the latest request is, i.e. `server.requests.length - 1`?

Reply:
Nice catch!